src/generator/AutoRest.Go/Templates/VersionTemplate.cshtml

```diff
@@ -24,8 +24,8 @@ import (
 @EmptyLine
 
 const (
-    major = "3"
-    minor = "1"
+    major = "4"
```

Comment:
It is @ [https://github.com/Azure/autorest/blob/6acdbca7a7e528af9a26f94b0b15ba12effc69c9/src/core/AutoRest.Core/Settings.cs#L237](https://github.com/Azure/autorest/blob/6acdbca7a7e528af9a26f94b0b15ba12effc69c9/src/core/AutoRest.Core/Settings.cs#L237). It'd be good when you do that please also update the documentations as well. 

---
In reply to: [77411287](https://github.com/Azure/autorest/pull/1397#discussion_r77411287) [](ancestors = 77411287)